File: package/main/src/Math/degToRad.ts
import { division } from "./division";
import { multiplication } from "./multiplication";
/**
 * Converts degrees to radians
 * @param  {number} x Angle in degrees
 * @returns number Angle in radians
 * @example degToRad(180); // 3.141592653589793
 */
export const degToRad = (x: number) => {
  return multiplication(x, division(Math.PI, 180));
};

File: package/umt_python/src/math/deg_to_rad.py
import math

from .division import division
from .multiplication import multiplication


def deg_to_rad(x: float) -> float:
    """
    Converts degrees to radians.

    Args:
        x: Angle in degrees.

    Returns:
        Angle in radians.

    Example:
        >>> round(deg_to_rad(180), 10)
        3.1415926536
    """
    return multiplication(x, division(math.pi, 180))
